src/main/java/oreo/task/Deadline.java

```diff
@@ -0,0 +1,40 @@
+package oreo.task;
+
+import oreo.datetime.TimeParser;
+
+public class Deadline extends Task {
+    private String date;
+
+    private String time;
+
+    public Deadline(String d, String date, String time) {
```

Comment:
Shouldn't the d be named into a more straightforward variable?

Reply:
Adding on here — I think that while I can inspect the superclass `Task` to figure out what `d` means, it's an extra step that might slow down others working on your code.